### ISSUE TITLE
Login fails at GET /api/v3/flows/executor/default-authentication-flow/ 

### ISSUE THREAD

**Issue by xcsdm** [goauthentik/authentik#7254]
**Describe the bug**
Authentication fails.  I've tried my own auth flow, which includes a passwordless option.  This log is from switching back to the default auth flow.

**To Reproduce**
1. Enter Username
2. Enter Password
3. Press Enter

**Expected behavior**
This stage would normally be to select the authenticator to use.  (Key or TOTP, in my case)

**Screenshots**

**Logs**
<details>
    <summary>Stacktrace from authentik</summary>

```
Traceback (most recent call last):
  File "/authentik/flows/views/executor.py", line 287, in get
    stage_response = self.current_stage_view.dispatch(request)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 222, in get
    challenges = self.get_device_challenges()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 157, in get_device_challenges
    user_devices = list(devices_for_user(self.get_pending_user()))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator/__init__.py", line 93, in devices_for_user
    yield from device_set
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 398, in __iter__
    self._fetch_all()
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/ak-root/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/django_prometheus/db/common.py", line 69, in execute
    return super().execute(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.11/site-packages/psycopg/cursor.py", line 737, in execute
    raise ex.with_traceback(None)
django.db.utils.ProgrammingError: relation "authentik_stages_authenticator_static_staticdevice" does not exist
LINE 1: ...tic_staticdevice"."throttling_failure_count" FROM "authentik...
                                                             ^
```
</details>

**Version and Deployment (please complete the following information):**
- authentik version: gh-next as of 10/21/2023
- Deployment: docker-compose

**Additional context**

**Comment by BeryJu** [comment]
should be fixed by https://github.com/goauthentik/authentik/pull/7244

**Comment by CallMeTerdFerguson** [comment]
I just upgraded to 2023.10.0 mainline via docker and the upgrade broke my instance with this same error. Should this fix be in that image? If so, it's not fixed. If it's not in the image, any way to fix my instance barring a rollback?

**Comment by jfreax** [comment]
I had to downgrade to version 2023.8.3 again due to this issue.

**Comment by BeryJu** [comment]
this will be fixed shortly with 2023.10.1
